homeassistant/components/switch/pilight.py

```diff
@@ -0,0 +1,122 @@
+"""
+Support for switching devices via pilight to on and off.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.pilight/
+"""
+import logging
+
+import homeassistant.components.pilight as pilight
+from homeassistant.components.switch import SwitchDevice
+
+DEPENDENCIES = ['pilight']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Setup the pilight platform."""
+    # Verify that pilight-daemon connection is present
+    if not pilight.CONNECTED:
```

Comment:
You don't need this. This method will not be called if the setup of the `pilight` component has failed.